Makefile

```diff
@@ -8,12 +8,16 @@ prereq:
 	@type convert >/dev/null 2>&1 || { echo "media-gfx/imagemagick with corefonts, svg and truetype required" >&2; exit 1; }; \
 		type xsltproc >/dev/null 2>&1 || { echo "dev-libs/libxslt is required" >&2; exit 1; }
 
+index:
+	@./search_index.py text.xml > documents.js
```

Comment:
Do we depend on python (and the deps I'm seeing in the script) to build it already? If not please update the prereq target.

Reply:
Python 2.7 is required by libxslt [already](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-libs/libxslt/libxslt-1.1.33-r1.ebuild#n25), and all of the python dependencies for the script are part of the standard library.